### ISSUE TITLE
Set Cookie Upgrade please

### ISSUE THREAD

**Issue by yeninesil** [php-curl-class/php-curl-class#413]
$cookie = $curl->responseCookies;

$curl->setCookieArray($cookie);

şeklinde bi kullanım yapın lütfen.

**Comment by zachborboa** [comment]
Hi. Thanks for the suggestion. Will implement `Curl::setCookies`.
```php
$cookie = $curl->responseCookies;
$curl->setCookies($cookie);
```

**Comment by Linuxhombre** [comment]
This would be simply amazing.  I'd be happy to battle-test a bit.

**Comment by zachborboa** [comment]
Fixed in #418.
